docs/eventlisteners.md

```diff
@@ -257,7 +264,60 @@ spec:
 
 With the help of `kubernetesResource` user can specify [PodTemplateSpec](https://github.com/kubernetes/api/blob/master/core/v1/types.go#L3704).
 
-Right now the allowed values as part of `podSpec` are
+#### CustomResource
+`CustomResource` have one field which supports dynamic object
+* runtime.RawExtension
+
+Here we will use a [Knative Service](https://knative.dev/docs/) as an example to demonstrate usage of `CustomResource`
+
+**Note:** `Knative Should be installed on the cluster` [ref](https://github.com/tektoncd/community/blob/main/teps/0008-support-knative-service-for-triggers-eventlistener-pod.md#note)
+```yaml
+spec:
+  resources:
+    customResource:
+      apiVersion: serving.knative.dev/v1
+      kind: Service
+#      metadata:
+#        name: knativeservice # name field is optional if not provided Triggers will use el name with the el- prefix ex: el-github-knative-listener
+      spec:
+        template:
+          spec:
+            serviceAccountName: tekton-triggers-example-sa
+            containers:
+            - resources:
+                requests:
+                  memory: "64Mi"
+                  cpu: "250m"
+                limits:
+                  memory: "128Mi"
+                  cpu: "500m"
+```
+
+With the help of `CustomResource` user can specify any dynamic object which follows below Contract
+
+##### Contract
+For Knative or new custom should satisfy [WithPod{}](https://github.com/knative/pkg/blob/master/apis/duck/v1/podspec_types.go#L41)
```

Comment:
What is "new custom"? Why is it "Knative or new custom"?

Reply:
Its new CRD instead of new Custom

Modified sentence